packages/gui/src/components/nfts/NFTContextualActions.tsx

```diff
@@ -53,12 +57,13 @@ export enum NFTContextualActionTypes {
   MoveToProfile = 1 << 2, // 4
   CancelUnconfirmedTransaction = 1 << 3, // 8
   Hide = 1 << 4,
-  Burn = 1 << 5,
-  CopyNFTId = 1 << 6, // 16
-  CopyURL = 1 << 7, // 32
-  ViewOnExplorer = 1 << 8, // 64
-  OpenInBrowser = 1 << 9, // 128
-  Download = 1 << 10, // 256
+  Invalidate = 1 << 5,
+  Burn = 1 << 6, // 16
+  CopyNFTId = 1 << 7, // 32
+  CopyURL = 1 << 8, // 64
+  ViewOnExplorer = 1 << 9, // 128
+  OpenInBrowser = 1 << 10, // 256
+  Download = 1 << 11, // 512
```

Comment:
Minor: the commented values have been wrong for a couple of releases now. Would be nice to fix it up.

```suggestion
  Hide = 1 << 4, // 16
  Invalidate = 1 << 5, // 32
  Burn = 1 << 6, // 64
  CopyNFTId = 1 << 7, // 128
  CopyURL = 1 << 8, // 256
  ViewOnExplorer = 1 << 9, // 512
  OpenInBrowser = 1 << 10, // 1024
  Download = 1 << 11, // 2048
```

Reply:
post-merge